### PR TITLE
Newsletter opt-out page per bug 1467252

### PR DESCRIPTION
**Pull request by peiying2**
German only

### CODE REVIEW

**Comment by flodolo** [comment]
> German only

Why would that be? This is a file exposed to all languages with newsletter.

File: en-US/newsletter/opt-out-confirmation.lang [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+;You’ll continue to receive other emails you’re subscribed to, along with occasional important updates about your account. To manage all your subscriptions, enter your email below — we need to make sure we’re talking to the right person.
+You’ll continue to receive other emails you’re subscribed to, along with occasional important updates about your account. To manage all your subscriptions, enter your email below — we need to make sure we’re talking to the right person.
+
+
+# Field of a form
+;Your email address:
+Your email address:
+
+
+# Do localize the example in your language.
```
Comment (flodolo) [inline en-US/newsletter/opt-out-confirmation.lang]: Quoting myself from IRC

```
add a comment to use @example.com though, and localize the name
```

@example.com must stay, they can localize "yourname".

File: en-US/newsletter/opt-out-confirmation.lang [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+;You’ll continue to receive other emails you’re subscribed to, along with occasional important updates about your account. To manage all your subscriptions, enter your email below — we need to make sure we’re talking to the right person.
+You’ll continue to receive other emails you’re subscribed to, along with occasional important updates about your account. To manage all your subscriptions, enter your email below — we need to make sure we’re talking to the right person.
+
+
+# Field of a form
+;Your email address:
+Your email address:
+
+
+# Only localize "yourname" and leave the rest as is.
```
Comment (flodolo) [inline en-US/newsletter/opt-out-confirmation.lang]: …and leave @example.com unchanged

**Comment by peiying2** [comment]
The project is for the "email" team thus limited to a few locales: pt-br, ru, id, pl, de, es, fr. This one is different from the newsletter locales defined in langchecker: https://github.com/mozilla-l10n/langchecker/blob/master/app/config/locales.inc.php#L55

**Pull request merged**